cmd/downloadcmd/s3.go

```
@@ -0,0 +1,51 @@
+	"github.com/spf13/cobra"
+)
+
+var (
+	bucket, prefix, dir string
+)
+
+func init() {
+	DownloadS3Cmd.Flags().StringVarP(&bucket, "bucket", "b", "", "AWS S3 bucket name that used for saving binlog files (required)")
+	DownloadS3Cmd.Flags().StringVarP(&prefix, "prefix", "p", "", "AWS S3 file prefix (folder) that used for saving binlog files (required)")
+	DownloadS3Cmd.Flags().StringVarP(&dir, "dir", "d", "", "A local directory that stores the objects (required)")
+	DownloadS3Cmd.MarkFlagRequired("bucket")
```

golangci-lint (1.64.8)

Error return value of `DownloadS3Cmd.MarkFlagRequired` is not checked

(errcheck)